src/core/Form/Combobox/ComboboxItem/ComboboxItem.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlLi } from '../../../../reset';
+import { CheckboxProps, Checkbox } from '../../Checkbox/Checkbox';
+import { baseStyles } from './ComboboxItem.baseStyles';
+
+const baseClassName = 'fi-combobox-item';
+
+const comboboxItemClassNames = {
+  wrapper: `${baseClassName}_wrapper`,
+  currentSelection: `${baseClassName}--currentSelection`,
+};
+
+export interface ComboboxItemProps
+  extends Omit<CheckboxProps, 'ref' | 'forwardedRef'> {
+  /** ComboboxItem container div class name for custom styling. */
+  className?: string;
+  currentSelection: boolean;
```

Comment:
Does this refer to active, hover or something completely different?

Reply:
It is the `true` when the keyboard focus is on that item. And that item is then styled based on that.

It could be renamed e.g `hasKeyboardFocus` to be more describing.

Reply:
It is now renamed as `hasKeyboardFocus`.